playbooks/ood-overrides-common.yml

```diff
@@ -97,4 +97,4 @@ pun_custom_env:
 # See https://osc.github.io/ood-documentation/latest/customization.html#pinning-applications-to-the-dashboard
 pinned_apps:
   - category: 'Interactive Apps'
-  - sys/shell/ssh/ondemand
+  - sys/shell
```

Comment:
Don't get your point as main is now tight to OOD 3.0. You may have to stay on the 1.0.37 version, as the next one will be pushing OOD 3.x

Reply:
well, we have many deployments with previous az-hop versions and may need some of the new features (e.g. naming of VM instances)

Reply:
in any case, if upgrade to OOD3 is smooth and the image is out, we can do that

Reply:
OOD 3.x works on the already published image, it's the other we need to confirm

Reply:
ok great!

Reply:
Just to be sure, you are referring to the marketplace image, right?

```
az vm image list --publisher azhpc --offer azhop-desktop --all --output table
Architecture    Offer          Publisher    Sku              Urn                                                 Version
--------------  -------------  -----------  ---------------  --------------------------------------------------  --------------
x64             azhop-desktop  azhpc        almalinux-8_7    azhpc:azhop-desktop:almalinux-8_7:2023.0612.1534    2023.0612.1534
x64             azhop-desktop  azhpc        almalinux-8_7    azhpc:azhop-desktop:almalinux-8_7:2023.0705.1621    2023.0705.1621
```

Just because the last image was published 5th of July, but this PR with the change to the desktop image was merged only in August

Reply:
It would be super useful to have access to a record of how these marketplace images were created (some git tag of some repository) - in that case we would not have to ask you

Reply:
yes, need to think about it

Reply:
> az vm image list --publisher azhpc --offer azhop-desktop --all --output table

@ltalirz yes the other way works. OOD2.x works when using the upcoming updated desktop image

Reply:
Ok, great!

When you say "upcoming", just to clarify:
Does this mean the currently published desktop image does not yet include the websockify upgrade done below (but still works with OOD 3)?

Reply:
yes this is correct. The existing image does works with OOD 2.x and 3.x. The next image update to be pushed will allow the same.